resources/js/Components/Country.vue

```diff
@@ -0,0 +1,73 @@
+<script setup>
+import { ref } from 'vue'
+import { router, useForm } from '@inertiajs/vue3'
+
+
+const props = defineProps({
+  country: Object,
+})
+
+function destroyCountry(countryId) {
+  router.delete(`/countries/${countryId}`)
+}
+
+function updateCountry(countryId) {
+  updateCountryForm.patch(`/countries/${countryId}`)
+  openEditWindow()
+}
+
+const updateCountryForm = useForm({
+  name: props.country.name,
+  alternativeName: props.country.alternativeName,
+  latitude: props.country.latitude,
+  longitude: props.country.longitude,
+  iso: props.country.iso,
+})
+
+const isEditWindowOpened = ref(false)
+
+function openEditWindow() {
+  isEditWindowOpened.value = !isEditWindowOpened.value
+}
+
+</script>
+
+<template>
+  <div class="m-2 flex flex-col rounded border bg-gray-50 p-2">
+    <div class="huge flat flags flex items-center space-x-2">
+      <i :class="country.iso" class="flat flag" />
+      <p>{{ country.name }}</p>
+    </div>
+
+    <div v-if="isEditWindowOpened" class="mt-2 w-full space-y-2">
+      <form class="flex w-1/2 flex-col space-y-2" @submit.prevent="updateCountry(country.id)">
+        <input v-model="updateCountryForm.name" class="border px-2 py-1" type="text" placeholder="Name" required>
+        <input v-model="updateCountryForm.alternativeName" class="border px-2 py-1" type="text" placeholder="Alternative name">
+        <input v-model="updateCountryForm.latitude" class="border px-2 py-1" type="text" placeholder="Latitude" required>
+        <input v-model="updateCountryForm.longitude" class="border px-2 py-1" type="text" placeholder="Longitude" required>
+        <input v-model="updateCountryForm.iso" class="border px-2 py-1" type="text" placeholder="ISO 3166" required>
+
+        <div class="flex justify-between">
+          <button v-show="isEditWindowOpened" type="submit" class="flex w-fit rounded border border-blue-500 bg-white px-5 py-2 text-white">
+            <img v-show="isEditWindowOpened" width="18" src="https://img.icons8.com/ios/50/3382f6/save--v1.png" alt="">
+          </button>
+
+          <button v-show="isEditWindowOpened" class="border-light flex w-fit items-center space-x-1 rounded-full border border-gray-500 bg-gray-100 px-2" @click="openEditWindow">
```

Comment:
Do we need this **v-shows**? We are already hiding it with **v-if** in **div** above.